ge09_team01/views/motorcycle_registry_inherit.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="motorcycle_registry_view_form_inherit_motorcycle_registry" model="ir.ui.view">
+        <field name="name">motorcycle.registry.view.form.inherit.motorcycle.registry</field>
+        <field name="model">motorcycle.registry</field>
+        <field name="inherit_id" ref="motorcycle_registry.motorcycle_registry_view_form" />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <field name="repair_order_ids" invisible="1"/>
+                <field name="count_orders" invisible="1"/>
+                    <button class="oe_stat_button"
+                        type="object"
+                        name="action_view_repair_order"
+                        string="Repair"
+                        title="Repair Order"
+                        icon="fa-gear"
+                        attrs="{'invisible': [('count_orders', '=', 0)]}">
+                        <field name="repair_order_ids" widget="statinfo" string="Repairs" />
+                    </button>
+            </div>
+        </field>
+    </record>
+</odoo>
```

Comment:
Missing newline. Try to install a vscode formatter for xml files, it will make your life way easier

Reply:
Same x)